specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/expressRouteCircuit.json

```diff
@@ -1463,11 +1463,11 @@
     "ExpressRouteCircuitConnectionPropertiesFormat": {
       "properties": {
         "expressRouteCircuitPeering": {
-          "$ref": "#/definitions/ExpressRouteCircuitPeering",
```

Comment:
@dedhar  I get a feeling you may have to verify/update examples with this change. PTAL

Reply:
actually the examples are already updated. That's how they found the issue. The examples were based on the PUT / GET / DELETE CAlls from example

Reply:
POSTMAN Examples  
PUT 
URI : https://management.azure.com/subscriptions/4bffbb15-d414-4874-a2e4-c548c6d45e2a/resourcegroups/ASH-Cust14/providers/Microsoft.Network/expressRouteCircuits/ASH-Cust14-ER/peerings/AzurePrivatePeering/connections/cktconnashsearepro?api-version=2017-03-01
Body
{
    "name": "cktconnashseaauth",
    "subscriptionId": "4bffbb15-d414-4874-a2e4-c548c6d45e2a",
      "properties": {
	    "expressroutecircuitpeering": {
			"id":"/subscriptions/4bffbb15-d414-4874-a2e4-c548c6d45e2a/resourceGroups/ASH-Cust14/providers/Microsoft.Network/expressRouteCircuits/ASH-Cust14-ER/peerings/AzurePrivatePeering"
		},
		"peerexpressroutecircuitpeering": {
			"id":"/subscriptions/4bffbb15-d414-4874-a2e4-c548c6d45e2a/resourceGroups/SEA-Cust14/providers/Microsoft.Network/expressRouteCircuits/SEA-Cust14-ER/peerings/AzurePrivatePeering"
		},
		"authorizationKey": "4e664b5d-e6f4-42d3-94c1-09c7b563fb97",
        "addressPrefix": "70.0.0.0/29"
      }
  }

DELETE
URI : https://management.azure.com/subscriptions/4bffbb15-d414-4874-a2e4-c548c6d45e2a/resourceGroups/ASH-Cust14/providers/Microsoft.Network/expressRouteCircuits/ASH-Cust14-ER/peerings/AzurePrivatePeering/connections/cktconnashseaauth?api-version=2017-03-01

Body : None

GET
URI : https://management.azure.com/subscriptions/99c33776-9f4e-4e58-abe8-9263db1b9c6e/resourceGroups/dedhareastusinit/providers/Microsoft.Network/expressRouteCircuits/dedharcktinit/peerings/AzurePrivatePeering/connections/test?api-version=2017-03-01
Body 
{
    "name": "test",
    "id": "/subscriptions/99c33776-9f4e-4e58-abe8-9263db1b9c6e/resourceGroups/dedhareastusinit/providers/Microsoft.Network/expressRouteCircuits/dedharcktinit/peerings/AzurePrivatePeering/connections/test",
    "etag": "W/\"47d8c5f9-fc6c-48b8-a8be-9cc25d765cff\"",
    "properties": {
        "provisioningState": "Succeeded",
        "expressRouteCircuitPeering": {
            "id": "/subscriptions/99c33776-9f4e-4e58-abe8-9263db1b9c6e/resourceGroups/dedhareastusinit/providers/Microsoft.Network/expressRouteCircuits/dedharcktinit/peerings/AzurePrivatePeering"
        },
        "peerExpressRouteCircuitPeering": {
            "id": "/subscriptions/8c992d64-fce9-426d-b278-85642dfeab03/resourceGroups/dedhareastuspeer/providers/Microsoft.Network/expressRouteCircuits/dedharcktpeer/peerings/AzurePrivatePeering"
        },
        "addressPrefix": "40.0.0.0/29",
        "authorizationKey": "3082022706092A864886F70D010703A082021830820214020100318201C0308201BC0201003081A330818B310B3009060355040613025553311330110603550408130A57617368696E67746F6E3110300E060355040713075265646D6F6E64311E301C060355040A13154D6963726F736F667420436F72706F726174696F6E31153013060355040B130C4D6963726F736F6674204954311E301C060355040313154D6963726F736F667420495420544C5320434120320213200000697FEE438283C85C8DA500000000697F300D06092A864886F70D01010105000482010046C7303437AD210105AC20B0547B0F81487E155055D2EB18F4203EC8BCA295946253758DEF6C1DCC7C5ECA2C411FC878A889D0C536AA3ED6E65D78C42B7DDF70058F5CD0F59F50EB19D89C9F39783BF7B3E2DF17C67516D2A02B28DDD588F036003A37EBB10FC9326405D6B18470471C28ACC84222C029C2FB979DC34C1C73399B99F0C3B78C282EE87A6399BF6442CDC6F12DFD76AC4B953E136D185CA6FC337BB2408ACF3F147FB498866594F5ED007312810C3D2D9883F8CE2112AC661578EC77BC05C59ED70077D2B85B5CD868377CCDF8A325FF84B7FE86FA817C490F2AC40C9AE784887BDF0EAD3B11FB7E618C9EF128F175A2AB596BE4518E6C63EAE6304B06092A864886F70D010701301406082A864886F70D03070408B485DDCC846C6EDF8028BF48A88AC371D060CF96A07FF4F23B5173253DBA59B17309777078F5F1CFF8BAABCE92B5ECEAE12D",
        "circuitConnectionStatus": "Connected"
    }
}